### PR TITLE
test: modular/executor pkg adds UTs

### PR DESCRIPTION
**Pull request by sysvm**
### Description

modular/executor pkg adds UTs.

### Rationale

Ensure SP code quality.

### Example

N/A

### Changes

Notable changes: 
* N/A

### CODE REVIEW

File: modular/executor/execute_task.go [hunk 1]
```diff
@@ -68,8 +68,10 @@ func (e *ExecuteModular) HandleSealObjectTask(ctx context.Context, task coretask
 	}
 	blsSig, err := bls.MultipleSignaturesFromBytes(task.GetSecondarySignatures())
 	if err != nil {
+		log.Info(1)
```
Comment (will-2012) [inline modular/executor/execute_task.go:71]: Remove debug log.

File: modular/executor/execute_task.go [hunk 2]
```diff
@@ -68,8 +68,10 @@ func (e *ExecuteModular) HandleSealObjectTask(ctx context.Context, task coretask
...
 		return
 	}
+	log.Info(2)
```
Comment (will-2012) [inline modular/executor/execute_task.go:74]: Ditto.

**Review by will-2012 — APPROVED** [review]

**Review by joeylichang — APPROVED** [review]

**Review by flywukong — APPROVED** [review]

File: modular/executor/execute_task.go (hunk 1 above)
Comment (sysvm) [reply modular/executor/execute_task.go:71]: fixed

File: modular/executor/execute_task.go (hunk 2 above)
Comment (sysvm) [reply modular/executor/execute_task.go:74]: fixed

**Pull request merged**